packages/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,7 @@
 
   <uses-sdk android:minSdkVersion="16" />
 
-  <application android:name=".MainApplication" android:allowBackup="true" android:fullBackupContent="@xml/backup_rules" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:label="@string/app_name" android:theme="@style/AppTheme" android:networkSecurityConfig="@xml/network_security_config">
+  <application android:name=".MainApplication" android:allowBackup="false" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:label="@string/app_name" android:theme="@style/AppTheme" android:networkSecurityConfig="@xml/network_security_config">
```

Comment:
How is backup related to this PR?

Reply:
I ran into an issue with the manifest merging, and it's actually something we should have already disabled for security reasons